code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityPropertiesTests.java

```diff
@@ -99,10 +98,22 @@ public void text_toXDMData_OnlyAdID() {
 		Map<String, Object> xdmMap = props.toXDMData(false);
 
 		// verify
-		assertEquals("test-ad-id", props.getAdId());
```

Comment:
I think we can keep this one to make sure the value is correct

Reply:
Updated test case to include the assetEquals for actual ad ID value